CRM/Upgrade/Incremental/sql/5.39.alpha1.mysql.tpl

```diff
@@ -10,5 +10,4 @@ CREATE TABLE `civicrm_translation` (
   `string` longtext NOT NULL COMMENT 'Translated string',
   PRIMARY KEY (`id`),
   INDEX `index_entity_lang`(entity_id, entity_table, language)
-)
-ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
+);
```

Comment:
@demeritcowboy do you think it makes any sense to leave in the ROW_FORMAT and ENGINE?

Reply:
InnoDB seems safe. To be honest I'm not sure if row_format=dynamic is safe if, let's say, you have just utf8 and don't have those settings like innodb_large_prefix etc. I don't have such a database handy to check, and internet searching doesn't seem to answer that question for me.

Reply:
Ok it seems I do have a (non-civi) mariadb 5.5 db running somewhere that has all the opposite settings (default=latin1, file-format = antelope, large-prefixes=OFF, file-per-table=OFF) so it's as good a test as any. If I make a dummy db and try running the create with innodb and row_format, it doesn't crash I just get two warnings:

```
| Warning | 1478 | InnoDB: ROW_FORMAT=DYNAMIC requires innodb_file_per_table. |
| Warning | 1478 | InnoDB: assuming ROW_FORMAT=COMPACT.  
```

Even if I change the db default to utf8 instead of latin it's the same. So that seems like a good indicator it would be safe.

Reply:
Maybe just re-add the ENGINE=INNDODB then just in case someone has a default of like MyISAM

Reply:
ah I see you have already done so